packages/swap/src/translators/reactjs/state/state.ts

```diff
@@ -376,16 +377,8 @@ const orderReducer = (
 
         if (draft.orderData.selectedPoolCalculation === undefined) break
 
-        if (
-          draft.orderData.amounts.sell.tokenId ===
-          draft.orderData.selectedPoolCalculation.pool.tokenA.tokenId
-        ) {
-          draft.orderData.amounts.buy =
-            draft.orderData.selectedPoolCalculation.sides.buy
-        } else {
-          draft.orderData.amounts.sell =
-            draft.orderData.selectedPoolCalculation.sides.sell
-        }
+        draft.orderData.amounts.buy =
+          draft.orderData.selectedPoolCalculation.sides.buy
```

Comment:
`sides` are already on the right side, and sell is locked (doesn't get updated), only buy needs update